fuel/app/classes/materia/api/v1.php

```diff
@@ -775,6 +775,17 @@ static public function question_set_get($inst_id, $play_id = null, $timestamp =
 		return $inst->qset;
 	}
 
+	static public function question_history_get($inst_id)
```

Comment:
Would prefer any new api endpoints to be added via fuel rest controllers, not sure if that's reasonable here or not.

Reply:
Add type hinting to function params and return